Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -167,7 +159,7 @@ def setupParallelPipelines(
 parallel clones
 
 println "About to print error messages"
-String errors = "\n" \\ This is so that the first line after "Error" in the console
+String errors = "\n" // This is so that the first line after "Error" in the console
```

Comment:
I think this will error out even there there are no errors.

Reply:
I have added a length check for results array